docs/assumptions.rst

```diff
@@ -28,22 +28,27 @@ Application Behavior
 ---------------------------
 Consistency Model
 ---------------------------
-In the first version of UnifyFS, lamination will be explicitly initiated by a
-UnifyFS API call. In subsequent versions, we will support implicit initiation
-of file lamination. Here, UnifyFS will determine a file to be laminated based
-on conditions, e.g., ``fsync`` or ``ioctl`` calls, or a time out on ``close``
-operations. As part of the UnifyFS project, we will investigate these implicit
-lamination conditions to determine the best way to enable lamination of files
-without explicit UnifyFS API calls being made by the application.
-
-In the first version of UnifyFS, eventually, a process declares the file to be
-laminated through a UnifyFS API call.
-After a file has been laminated, the contents of the file cannot be changed.
-The file becomes permanently read-only.
-After lamination, any process may freely read any part of the file.
-If the application process group fails before a file has been laminated,
-UnifyFS may delete the file.
-An application can delete a laminated file.
+One key aspect of UnifyFS is the idea of "laminating" a file.  After a file is
+laminated, it's becomes "set in stone" and its data is accessible across all the
+nodes.  Laminated files are read-only, and cannot be modified in any way (but
+can be deleted).
+
+The typical use case is to laminate your checkpoint files after they've been
+written.  To laminate a file, simply call chmod() to remove all the write bits
+on the file.  If you have written to the file, you will also need to sync
```

Comment:
Let's write this part to be process-centric so it's clear that any process that writes to a file must also fsync, like "If a process has written to the file, it must also sync ..."